src/DynamoCore/Models/AnnotationModel.cs

```diff
@@ -120,7 +123,7 @@ public Double TextBlockHeight
             {
                 textBlockHeight = value;                
                 Y = InitialTop - textBlockHeight;
-                Height = InitialHeight + textBlockHeight;
+                Height = InitialHeight + textBlockHeight - minHeight;
             }
```

Comment:
I'm not totally sure, but it looks like we're using this to establish a margin around the boundary of the `AnnotationModel`, such a detail probably belongs on the `ViewModel` layer or, better yet, on the `View` layer.  Then it could be configured by recovering view specific configurations there.